tutorials/tutorial9_simple_3ratios_gearbox.py

```diff
@@ -171,13 +173,28 @@ def __init__(self, gearbox: GearBox, wltp_cycle: WLTPCycle,
         self.fuel_consumptions = fuel_consumptions
         self.gears_ratios = gears_ratios
         DessiaObject.__init__(self, name=name)
- 
+
+    @property
+    def cycle_time(self):
+        return [i+1 for i in range(len(self.wltp_cycle.cycle_speeds[:-1]))]

```

Comment:
Why did you turn this into a property ?

Reply:
I made this choice because it is a value that we use several times, so instead of calculating it in each function I chose to put it in property (encapsulate)

Reply:
It's only used twice, no ?
If you think it's worth the "complexity" it brings, i'm fine with it. But i'm not sure every user going through the tutorials will know what a @property is.